src/GameCards.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import {
+  faChild,
+  faHourglass,
+  faCommentSlash,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const GameCards = ({ games }) => {
+  return (
+    <div className="row row-cols-1 row-cols-md-3 g-4 mb-4">
+      {games.map((game) => {
+        return (
+          <div className="col">
```

Comment:
```suggestion
          <div key={game.name} className="col">
```

Reply:
D,'oh - thanks for catching that! I added the `col` after the fact.